config/deploy/docker/assets/Dockerfile.open-path-warehouse.pre-cache

```diff
@@ -1,112 +0,0 @@
-#########################################################
-## stage build for the self-signed certificate we need ##
-#########################################################
-FROM ruby:3.1.4-alpine3.18 AS cert
-
-RUN apk add --no-cache \
-  openssl
-
-WORKDIR /
-
-RUN openssl req -new -newkey rsa:4096 -days 3650 -nodes -x509 \
-    -subj "/C=US/ST=Vermont/L=Brattleboro/O=greenriver/CN=www.example.com" \
-    -keyout key.pem -out cert.pem
-
-########################
-## Stage for the code ##
-########################
-
-FROM ruby:3.1.4-alpine3.18 AS code
-
-RUN mkdir -p /app \
-    /app/app/assets /app/tmp/pids /app/lib/assets /app/vendor/assets \
-    /app/public /app/locale /app/var  \
-    /app/shape_files/CoC /app/shape_files/zip_codes.census.2018 \
-    /app/shape_files/block_groups /app/shape_files/states /app/shape_files/counties
-
-WORKDIR /app
-
-COPY Gemfile Gemfile.lock Rakefile config.ru package.json yarn.lock ./
-COPY bin ./bin
-COPY public ./public
-COPY lib ./lib
-COPY config ./config
-COPY app ./app
-COPY vendor ./vendor
-COPY config/deploy/docker/assets/dotenv.temporary ./.env
-COPY sample.irbrc ./.irbrc
```

Comment:
@yakloinsteak Looks like this got missed as well.  Any thoughts on where best to put this?

Reply:
https://github.com/greenriver/hmis-warehouse/pull/4004 Do we need the .env too?

Reply:
Not as far as I can tell.  I think that had something to do with assets maybe?